.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
```

Comment:
I'd suggest to put everything that the user is supposed to delete in HTML comments, e. g. everything besides the headings.
People tend to keep the descriptive template text and just add their text below it which greatly decreases readability.
If the explanations / templates are in HTML comments, they are not visible even if the user didn't remove them.